src/features/search-bar/organisms/search-bar.tsx

```diff
@@ -16,12 +18,12 @@ interface SearchbarProps {
   newCardHref: string;
 }
 
-// TODO: вынести из Searchbar логику с пользователем
 export const Searchbar: React.FC<SearchbarProps> = ({
   logoHref,
   newCardHref,
 }) => {
   useSearchQueryChanged();
+  const userInfo = useStore($currentUser);
```

Comment:
[`nitpick(non-blocking):`](https://conventionalcomments.org/#labels) Мб заменим просто на `viewer`?

Reply:
@azinit 
`question:` имеешь ввиду так?

```suggestion
  const viewer = useStore($currentUser);
```